pkg/frontend/installversions_list.go

```diff
@@ -0,0 +1,31 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+	"github.com/Azure/ARO-RP/pkg/util/version"
+)
+
+func (f *frontend) listInstallableVersions(w http.ResponseWriter, r *http.Request) {
+	log := r.Context().Value(middleware.ContextKeyLog).(*logrus.Entry)
+	versions := f.getInstallVersions()
+	b, err := json.Marshal(versions)
+	reply(log, w, nil, b, err)
+}
+
+// Creating this separate method so that it can be reused while doing some other stuff like validating the install version against available version.
+func (f *frontend) getInstallVersions() []string {
```

Comment:
nit: should this be `listInstallVersions` or `doListInstallableVersions`/`_listInstallableVersions` to remain consistent?

Reply:
Okay .. I will do it .. I thought of keeping it generic so that it can be used else where as well but still 'list' word will also do the work.